src/referrals-queue/referrals-queue.resource.ts

```diff
@@ -1,8 +1,13 @@
 import { openmrsObservableFetch } from "@openmrs/esm-api";
 import { map } from "rxjs/operators";
 
-export function getReferrals(): Observable<Object[]> {
+export function getReferrals({
+  fromDate,
+  toDate,
+  patient = null
+}): Observable<Object[]> {
   return openmrsObservableFetch(
-    `/ws/rest/v1/reportingrest/reportdata/cd7dfde7-764a-4da6-81c2-d5887ed1df51?startDate=2020-01-01&endDate=2020-04-01&locale=en`
+    `/ws/rest/v1/reportingrest/reportdata/cd7dfde7-764a-4da6-81c2-d5887ed1df51?startDate=${fromDate}&endDate=${toDate}&locale=en` +
```

Comment:
Minor tangential point while I'm thinking of it, let's just extract that UUID out into a constant... find for it just to stay in this file or in a stand-alone constants file, I don't have strong preference.

Also, let's not forget to pass in the locale at some point instead of hardcoding to "en"  (I don't expect it to be done now, but just throwing it out there while I notied it because it's something we are likely to forget, so just trying to keep it on our radar)

Reply:
Oh yeah, good idea about the constant! I'll make it a config variable. And yes, I have a `test.todo` about the locale.